### ISSUE TITLE
SwiftyJSON 5.0.2: EXCLUDED_ARCHS arm64

### ISSUE THREAD

**Issue by pmanuelli** [SwiftyJSON/SwiftyJSON#1159]
Hello, thank you for adding the AppPrivacy file to SwiftyJSON. 
I tried to use it but I encountered the following issue:

### What did you do?

I updated from SwiftyJSON 5.0.1 to SwiftyJSON 5.0.2. Run pod install and tried to compile my project

### What did you expect to happen?

My project to compile.

### What actually happened instead?

After running `pod install` I got this warning:

`[!] Can't merge user_target_xcconfig for pod targets: ["SwiftyJSON", "FBAudienceNetwork"]. Singular build setting EXCLUDED_ARCHS[sdk=iphonesimulator*] has different values.`

And when I try to run the project on a simulator with an Apple Silicon Mac I get this error:

`#error unsupported Swift architecture` on `SwiftyJSON-Swift` file.

I've compared the SwiftyJSON.podspec file from 5.0.1 and 5.0.2 and can't find anything related to EXCLUDED_ARCHS configuration. But when I look into the build settings on Xcode I see this:

<img width="919" alt="swiftyjson" src="https://github.com/SwiftyJSON/SwiftyJSON/assets/8539386/2f837892-e9dc-44d8-b8a8-73cb20cddab3">

If the arm64 architecture is excluded on simulators, it will not work. I had no problem with version 5.0.1 using the same environment.

I don't think that FBAudienceNetwork has anything to do with this. It just helped me to find this because the EXCLUDED_ARCHS can't be merged.

### Environment

List the software versions you're using:

 - SwiftyJSON: *5.0.2*
 - Xcode Version: *15.2 (15C500b)*
 - Swift Version: *Xcode Default*

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

 - Cocoapods: *1.15.2*

### Project that demonstrates the issue

[Sample project](https://drive.google.com/file/d/19tUT2eBnJAvZ5w5eLB7yypMzRrMXRYYZ/view?usp=sharing)

**Comment by wongzigii** [comment]
Try run pod repo update and then pod install again.

**Comment by pmanuelli** [comment]
Hello @wongzigii.

I did it and the result is the same.

I've updated my original report with more information. I'm trying to run the project on a simulator using an Apple Silicon Mac. Since on SwiftyJSON 5.0.2 arm64 is excluded from simulators, it fails when I try to run the application.

This was not an issue with SwiftyJSON 5.0.1 using the same environment.

**Comment by pmanuelli** [comment]
Ok, I got it. 

Seems like the SwiftyJSON.podspec of version 5.0.2 was modified after the file was published. That can lead to this kind of problems. Locally, on my spec repo I had [the original version of 5.0.2](https://github.com/CocoaPods/Specs/commit/f638b969f5d3d854a310c42d950e1e75ff12aa2f) where the architecture is excluded.

To fix this, I had to delete the local podspec file. If anyone has the same problem, to find and delete the file run `find ~/.cocoapods -type d -name "FairBidSDK" -exec rm -rf {} +` and then `pod install --repo-update`.

The warning is gone now. Thank you
